openff/bespokefit/tests/cli/test_worker.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from openff.bespokefit.cli.worker import worker_cli
+from openff.bespokefit.executor.utilities import celery
+
+
+@pytest.mark.parametrize(
+    "worker_type",
+    [
+        pytest.param("fragmenter", id="fragmenter"),
+        pytest.param("qc-compute", id="qc-compute"),
+        pytest.param("optimizer", id="optimizer"),
+    ],
+)
+def test_launch_worker(worker_type, runner, monkeypatch):
+    """Test launching a worker of the correct type, note we do not start the worker this is tested
+    in test_celery/test_spawn_worker
+    """
+
+    def mock_spawn(*args):
+        return True
+
+    monkeypatch.setattr(celery, "_spawn_worker", mock_spawn)
```

Comment:
We should look into swapping to [pytest-mock](https://github.com/pytest-dev/pytest-mock) at some point - it's essentially a much nicer to use version of monkeypatch with extra features!